### PR TITLE
Very small deco note

### PR DESCRIPTION
**Pull request by gnizla**

### CODE REVIEW

**Review by thea-m — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml [hunk 1]
```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
-                       <!-- <decoDesc>
+                        <decoDesc>
+                            <summary>
+                              This psalter contains iconic portraits of figures from the Old and New Testaments.
+                              It is among the most extensively illustrated examples from the mid-fifteenth century.
+                              Most miniatures take up a full page, but some ½ or less (cf. fols 4v, 146v).
+                              In all likelihood, this manuscript was scribed and decorated by Gabra Krǝstos,
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethd19.xml:232]: mark up with PRS13458GabraK?

**Comment by thea-m** [comment]
Thank you, wonderful! It's technically fine to keep the same PR, so no need to change this one, but in the future I think opening a new branch and individual PR per manuscript would contribute to reduce potential synchronisation problems.
Regarding the mark-up of persons: You are completely right, in order to be indexed, they need to be marked up. I believe we had agreed previously not to mark up protagonists in texts. If we decide to mark up and index protagonists of illustrations (and from my point of view, this decisions is completely yours as the expert) that would be a small inconsistency we would need to explain in the introduction.

**Review by thea-m — CHANGES_REQUESTED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml [hunk 2]
```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
-                       <!-- <decoDesc>
+                        <decoDesc>
+                            <summary>
+                              This psalter contains iconic portraits of figures from the Old and New Testaments.
+                              It is among the most extensively illustrated examples from the mid-fifteenth century.
+                              Most miniatures take up a full page, but some ½ or less (cf. fols 4v, 146v).
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethd19.xml:231]: what about `<locus target="#4v #146v"/>`?

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml [hunk 3]
```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                  <persName ref="PRS5741Johnthe">John the Baptist</persName> and
+                                  <persName ref="PRS11326Andrew">Andrew the Apostle</persName>,
+                                  <persName ref="PRS5666Jamesth"> James brother of the Lord</persName> and
+                                  <persName ref="PRS6751Mark">Mark</persName> with hand crosses and monastic hoods,
+                                  <persName ref="PRS2481Bartholo">Bartholomew</persName> and
+                                  <persName ref="PRS6902Matthew">Matthew</persName>,
+                                  <persName ref="PRS9496Thomas">Thomas</persName> and
+                                  <persName ref="PRS7843Philip">Philip</persName>.
+
+                                  <q xml:lang="gez"><persName ref="PRS5684JesusCh">ግዚእነ፡</persName> ዘከመ፡ ሰዐሞ፡
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethd19.xml:401]: Is the first እ in እግዚእነ missing?

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml [hunk 4]
```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                  <q xml:lang="gez"><persName ref="PRS3337Daniel">ዳንኤል፡</persName> ነቢይ፡</q>
+                                  <q xml:lang="gez"><persName ref="PRS5680Jeremiah">ኤርምያስ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS4400Gad">ጋድ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS13685Nathan">ናታን</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3814Enoch">ሄኖክ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3721Elijah">ኤልያስ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3723Elisha">ኤልሳዕ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS5773Joshua">ኢያሱ፡ መስፍን።</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS5584Isaiah">ኢሳይያስ፡</persName></q>
+                                  <q xml:lang="gez">ዘሩ[…]ል።</q>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethd19.xml:300]: Is the portion of text marked with [...] illegible due to faded ink or material damage?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Jacopo, I have very few questions mostly concerning the encoding

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml (hunk 3 above)
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml:401]: Thanks!

**Comment by gnizla** [comment]
> Thank you, wonderful! It's technically fine to keep the same PR, so no need to change this one, but in the future I think opening a new branch and individual PR per manuscript would contribute to reduce potential synchronisation problems. Regarding the mark-up of persons: You are completely right, in order to be indexed, they need to be marked up. I believe we had agreed previously not to mark up protagonists in texts. If we decide to mark up and index protagonists of illustrations (and from my point of view, this decisions is completely yours as the expert) that would be a small inconsistency we would need to explain in the introduction.

Thanks @thea-m 

Regarding mark up of persons: since there are captions identifying each individual figure, which is something that does not occur in most of the other illustrations, I identified them in this way. This is consistent with how I encoded other miniatures. I will not do it if the figures are not identified.

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml (hunk 4 above)
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml:300]: Damage

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml (hunk 1 above)
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml:232]: Should I mark up a hypothetical though?

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml (hunk 4 above)
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml]: Good, thank you. I took the freedom to encode it as `<gap reason="lost"/>`

File: OxfordBodleian/Juel-Jensen/BDLaethd19.xml (hunk 1 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml]: Since you state the probability very clearly verbally, I think it is appropriate here.

Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethd19.xml]: Thanks, done!

**Review by MassimoVilla — APPROVED** [review]

**Review by ElianaDalSasso — COMMENTED** [review]
Thank you Jacopo! I see that the PR of d. 17 is linked to a previous one, so just discard my other comment.

**Pull request merged**